lib/bulk.js

```diff
@@ -0,0 +1,157 @@
+module.exports = function Bulk(collection, ordered) {
+  var ops = [];
+  var executedOps = [];
+  var interface = {
```

Comment:
can we move away from using `interface` as a variable name to stay compatible with people who run in strict mode?

Reply:
#59 lists a solution of a readable abbreviation.